tests/lib/rule-tester/rule-tester.js

```diff
@@ -67,7 +67,7 @@ describe("RuleTester", () => {
     let ruleTester;
 
     /**
-     * @description A helper function to verify Node.js core error messages.
+     * A helper function to verify Node.js core error messages.
```

Comment:
The default configuration doesn't check for `@description` tags. We can `descriptionStyle: "any"` to allow both styles, but since this is the only instance in our codebase, it seemed like a good idea to just go with the default behavior. Happy to change it if others disagree, though.